.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish:
+    name: Build and publish to PyPI
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
```

Comment:
Just a possible improvement (to be "up-to-date" longer).

We could probably use [actions/setup-python@v2](https://github.com/marketplace/actions/setup-python) without any change needed.

FFTR